Tools/CISupport/ews-build/steps.py

```diff
@@ -2599,7 +2599,7 @@ class RunWebKitTests(shell.Test):
     test_failures_log_name = 'test-failures'
     ENABLE_GUARD_MALLOC = False
     EXIT_AFTER_FAILURES = '30'
-    command = ['python', 'Tools/Scripts/run-webkit-tests',
+    command = ['python3', 'Tools/Scripts/run-webkit-tests',
```

Comment:
Is this tested on uat instance? If not, can you please do that.
Also, is it working fine on build.webkit.org?

Reply:
No, but "testing" this on the UAT instance is not a great idea. We made this change a few months back, our problem was with iOS Simulators running out their available file handles. In order to test that failure mode, we would need to move iOS Simulator queues to the staging instance. The trouble is, that particular queue is backed up pretty badly and can't really spare the hardware (especially because we need both a builder and a tester)

I consulted @Technose and @robert-jenner last week about the right way to verify this change, which is why we did build.webkit.org first. You can see in https://build.webkit.org/#/builders/Apple-iOS-15-Simulator-Release-WK2-Tests?numbuilds=50 that we're running well. We have periodic problems booting the simulators, but that is a long-standing issue we saw with Python 3. Late last week, I changed the shebang to `python3`, so now everything except EWS is using Python 3.